group.py

```diff
@@ -2,4 +2,12 @@
 
 # Your code to go here...
 
-my_group =
+my_group = { "Jill" : {"age" : 26, "job" : "biologist", "connection" : ["Zalika", "John"]}, 
+           
+            "Zaklika" : {"age" : 28, "job" : "artist", "connection" : ["Jill"]},
+           
+            "John" : {"age" : 27, "job" : "writer", "connection" : ["Jill"]},
+           
+           "Nash" : {"age" : 34, "job" : "chef", "connection" : ["Zalika", "John"]}}
+
+
```

Comment:
[Suggestions] Perhaps add a list for the job key value so it can be empty (to represent people without a job). Also, it's good that you added a list for the connection because it can be empty, but you could use a dictionary within this list to give the the type of relation for each connection (to make the connections reciprocal). Well done, though!